### PR TITLE
[map] leaf pages

### PR DESCRIPTION
**Pull request by chejennifer**
- For leaf pages, zoom in on the place of interest 

dev has been updated (http://dev.datacommons.org/tools/map#%26sv%3DCumulativeCount_MedicalConditionIncident_COVID_19_PatientDeceased%26pc%3D1)
Screen Shot 2021-06-16 at 10 42 04 PM
Screen Shot 2021-06-16 at 10 42 21 PM

### CODE REVIEW

File: static/js/chart/draw_choropleth.ts [hunk 1]
```diff
@@ -98,23 +104,46 @@ function drawChoropleth(
...
+    if (geoJsonFeature) {
+      fitSize(
+        chartWidth - legendWidth,
+        chartHeight,
+        geoJsonFeature,
+        projection,
+        geomap,
+        ZOOMED_SCALE_AMOUNT
+      );
+    }
```
Comment (beets) [inline static/js/chart/draw_choropleth.ts:136]: how should we handle the case where a feature wasn't found?

**Review by beets — COMMENTED** [review]
lg! just one error handling case to handle..

File: static/js/chart/draw_choropleth.ts (hunk 1 above)
Comment (chejennifer) [reply static/js/chart/draw_choropleth.ts:136]: I guess when a feature isn't found, we can just fit it like we were drawing the map without zooming in?

**Comment by beets** [comment]
yeah, that sounds good.

File: static/js/chart/draw_choropleth.ts (hunk 1 above)
Comment (chejennifer) [reply static/js/chart/draw_choropleth.ts:136]: Or maybe we could have an error message that says "Sorry, unable to draw the map for this place"

**Comment by beets** [comment]
either way works. we can start with an error message - that would be less
confusing when it does happen.

File: static/js/chart/draw_choropleth.ts (hunk 1 above)
Comment (chejennifer) [reply static/js/chart/draw_choropleth.ts:136]: or we can have an error message that says something like "Sorry, unable to draw map for this place"

Comment (chejennifer) [reply static/js/chart/draw_choropleth.ts:138]: done

**Review by beets — APPROVED** [review]
lg - thanks for the update

**Pull request merged**